flycheck.el

```diff
@@ -4257,6 +4263,27 @@ universal prefix arg, and only the id with normal prefix arg."
     (`(eval ,_) t)
     (_ nil)))
 
+(defun flycheck-default-directory-wrapper (checker)
+  "Function returning apropriate current working directory.
```

Comment:
Please use imperative style for the first line of a docstring.  Take a look at other docstrings to see the style that we use.